content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -44,7 +44,7 @@ that provides a set of stateless replicas.
 ## Limitations
 
 * The storage for a given Pod must either be provisioned by a
-  [PersistentVolume Provisioner](https://github.com/kubernetes/examples/tree/master/staging/persistent-volume-provisioning/README.md)
+  [PersistentVolume Provisioner](/docs/concepts/storage/dynamic-provisioning/) ([examples here](https://github.com/kubernetes/examples/tree/master/staging/persistent-volume-provisioning/README.md))
   based on the requested `storage class`, or pre-provisioned by an admin.
```

Comment:
Let's align to our style guide here so the rest of the file can also follow it:
```suggestion
  based on the requested _storage class_, or pre-provisioned by an admin.
```

Reply:
Why italics here instead of camel case?

Reply:
In this location, we're introducing the concept, not the API kind.